packages/fast-components-styles-msft/src/utilities/high-contrast.ts

```diff
@@ -94,6 +105,15 @@ export const highContrastDoubleFocus: CSSRules<DesignSystem> = {
 
 // Used to set 'selected-text' color
 export const highContrastSelection: CSSRules<DesignSystem> = {
+    [highContrastSelector]: {
+        background: HighContrastColor.selectedBackground,
+        color: HighContrastColor.selectedText,
+        fill: HighContrastColor.selectedText,
+    },
+};
+
+// Used to set 'selected-text' color with an outline
+export const highContrastSelectionOutline: CSSRules<DesignSystem> = {
```

Comment:
Looks like I missed one last time, but "selection" should be "selected". Can we rename the new ones and deprecate/alias the other since it's already been published. "Selection" sounds more like highlighted text or a single item in a list. "Selected" is a state like we're using it here. That will coordinate better with a new feature I'm about to do for non-HC selected colors.

Reply:
updated to be `selected`